src/Analyzer.Cli/CommandLineParser.cs

```diff
@@ -96,13 +128,90 @@ private void RootCommandHandler(FileInfo templateFilePath, FileInfo parametersFi
                 }
 
                 Console.WriteLine($"{IndentedNewLine}Rules passed: {passedEvaluations}");
+
+                return 1;
             }
             catch (Exception exp)
             {
                 Console.WriteLine($"An exception occured: {GetAllExceptionMessages(exp)}");
+                return -1;
             }
         }
 
+        private void AnalyzeDirectory(DirectoryInfo directoryPath)
+        {
+            try {
```

Comment:
minor: put the bracket on its own line (also for `if` statement on line 175), and remove the extra blank line.